app/views/users/_private_user_data.html.erb

```diff
@@ -0,0 +1,22 @@
+<%- model_class = User -%>
+
+<h4><%=t('.private_data')%></h4><br>
+
+<% [:email, :zip_code, :city, :street].each do |field| %>
+	<% unless @user[field].blank? %>
+		<p><strong><%= model_class.human_attribute_name(field)%>: </strong><%= @user[field] %></p>
+	<% end %>
+<% end %>
+
+<% if @user.is_hiwi? %>
+	<p><strong><%= t('.user_status')%>: </strong><%= t('.hiwi') %></p>
```

Comment:
Laut dem Test von meinem PO gibt es auch Fälle wo ein user weder hiwi noch wimi ist. In dem Fall soll das Feld "Status" nicht angezeigt werden.
...<%= @user.is_hiwi? ? t('.hiwi') : t('.wimi') %>...
ist also nicht ausreichend.